trino/mapper.py

```diff
@@ -31,6 +31,29 @@ def map(self, value: Any) -> Optional[T]:
         pass
 
 
+class BooleanValueMapper(ValueMapper[bool]):
+    def map(self, value: Any) -> Optional[bool]:
+        if value is None:
+            return None
+        if isinstance(value, bool):
+            return value
+        if str(value).lower() == 'true':
+            return True
+        if str(value).lower() == 'false':
+            return False
```

Comment:
is there a way to avoid this and return map keys as boolean instead?

Reply:
Technically yes, but it's much harder to do on server. There's already some code `FixJsonDataUtils` which tries to handle some of this but it doesn't work well or is easy to do for structural types.